CONTRIBUTING.md

```diff
@@ -1,65 +1,43 @@
 
 Note that service-specific types (such as “Twitter Privacy Policy”) are not allowed. Terms types aim at allowing comparisons across services and should thus be generic.
 
-### Provide examples
-
-Provide at least the terms you intend to track as example of the new terms type, with relevant information on its source and context:
-
-- Include a direct link to the source document.
-- Name the service provider and describe its service.
-- Provider-given terms name: provide the title or name given to these terms by the service provider.
-
-### Add references
-
-Include any relevant [references](./README.md#references) that may help in understanding the purpose of this type. This could include legal definitions, discussions, or any other resource that provides context or background information.
-
-References will then be listed in the type definition, with their URL and a title describing the content and source.
-
-### Build consensus
-
-Publicise your suggestion across Open Terms Archive channels and engage the community. Throughout the discussion, the name and the triptych should be challenged, taking into account international perspectives on phrasings and the variety of definitions across jurisdictions.
-
-In order to ensure global relevance, your suggestion will only be considered for inclusion when there are at least three examples of such terms, each from different service providers. You are welcome to provide several examples on your own.
+### Integration process
 
-### Integration
+New suggestions for terms types are categorized as `Submitted`. The Core Team reviews these suggestions and ensures that all the necessary information is gathered and clear and that at least 5 examples are provided, maximizing industry and jurisdiction diversity. When this is the case, the discussion is moved to the `Qualified` category and opened up to the community.
 
-Once:
+Once your suggestion is `Qualified`, build consensus by publicizing it across Open Terms Archive channels and engaging with the community. Throughout the discussion, the data should be critically examined, especially taking into account international perspectives on vocabulary and the variety of definitions across jurisdictions.
 
-- at least three examples from different service providers have been given;
-- a consensus has been reached on the triptych;
-- a consensus has been reached on the name;
-- at least 2 weeks have elapsed since the opening of the discussion, to ensure visibility throughout the community.
+Once a consensus has been reached and at least 2 weeks have passed since the discussion was opened, to ensure visibility within the community, a pull request for the addition to the database can be submitted. To facilitate tracking and participation, each pull request should contain only one type addition and must link to the corresponding discussion.
```

Comment:
```suggestion
Once a consensus has been reached and at least 2 weeks have passed since the discussion was opened (to ensure visibility within the community), a pull request for the addition to the database can be submitted. To facilitate tracking and participation, each pull request should contain only one type addition and must link to the corresponding discussion.
```